KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -127,8 +127,8 @@ public boolean removeReferrer(ILiveEntryReferrer obj) {
 		}
 
 		public LiveEntryCache(KalturaLiveEntry liveEntry, KalturaMediaServerIndex serverIndex, String applicationName) {
-			this(liveEntry);
-			register(serverIndex, applicationName);
+			// Check if this constructor is even relevant? Might be possible to delete?
```

Comment:
You removed this when we were sitting together. But nonetheless, these two lines are not relevant as we don't call "register" from here and I'm pretty sure this constructor is not being used at all.